llvm/test/Feature/Repo/repo_alias_aliasOfAlias.ll

```diff
@@ -0,0 +1,25 @@
+; Check the pruning for an alias (_ZN7derivedD1Ev) whose aliasee is another alias (_ZN7derivedD2Ev).
+;
+; RUN: rm -f %t.db
+; RUN: env REPOFILE=%t.db opt -S -mtriple x86_64-pc-linux-gnu-repo %s -o %t
+; RUN: env REPOFILE=%t.db llc -filetype=obj %t
+; RUN: env REPOFILE=%t.db opt -S -mtriple x86_64-pc-linux-gnu-repo %s | FileCheck %s
+
+target triple = "x86_64-pc-linux-gnu-repo"
+
+%class.base = type { i8 }
+%class.derived = type { i8 }
```

Comment:
Could we make this test a little easier to digest? I don’t think that we do anything with types `%class.base` or `%class.derived`. There are two pairs of function names (`_ZN4baseD?Ev` and `_ZN7derivedD?Ev`) each rather long with their IA64 ABI naming decoration (which we don’t need) and differing only by one character. Even `_ZN4baseD2Ev` doesn’t need it’s `this` argument for the purposes of the test.